talkative-tangs/Bangazon-LLC-2
# Bn add to cart

Link to Ticket
Addresses but does not complete #2 #6 #10 

Description of Proposed Changes
Steps to Test
Outline the steps to test
When logged in, user can add product to open order or create new order
User can delete item from order detail- *page currently redirects to index, should redirect back to order detail.

`git fetch --all`
`git checkout bn-add-to-cart`

The Order model has changed so you will need to remigrate:

`python manage.py migrate`
`python manage.py makemigrations website`
`python manage.py migrate`
`python loaddata db.json`

Impacted Areas in Application:
views.py
urls.py
product_detail.html

Team to review:
@laboyd001 
@ousamasama 
@CurtainUp
